tt_eager/tt_dnn/op_library/groupnorm/groupnorm_op.hpp

```diff
@@ -38,4 +38,45 @@ Tensor groupnorm(
     const MemoryConfig& mem_config = operation::DEFAULT_OUTPUT_MEMORY_CONFIG);
 
 }  // namespace tt_metal
+
+
+namespace operations {
+
+    using namespace tt_metal;
+namespace primary {
+struct GroupNormShardedMultiCoreProgramConfig {
+    CoreCoord compute_with_storage_grid_size;
+    MathFidelity math_fidelity;
+    DataType im_data_format;
```

Comment:
`im_data_format` means `image_data_format` or is this a typo and should be `in_data_format`?

Reply:
it is the intermediate data format, for the intermediate cbs